src/displayapp/screens/Calculator.h

```diff
@@ -0,0 +1,40 @@
+
+#pragma once
+
+#include <array>
+#include "Screen.h"
+#include "../LittleVgl.h"
```

Comment:
These includes should be non relative now. https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/doc/coding-convention.md#coding-style

Reply:
Another relic from when most of this was written. Will fix.